### PR TITLE
Get the basic outline of this website set up 

### PR DESCRIPTION
**Pull request by cansavvy**
Here I'm setting up a basic outline for this website based on what was discussed in Slack and BaseCamp. 

I've mostly filled out the Training Resources page, but the rest needs more filling out.

### CODE REVIEW

File: _data/navigation.yml [hunk 1]
```diff
@@ -1,32 +1,36 @@
...
+  - title: "Hutch Data"
+    url: /hutch_data/
+  - title: "Training Resources"
+    url: /training_resources/
+  - title: "Community"
     url: /community/
-  - title: "News"
-    url: /news/
+  - title: "Contact us"
+    url: https://forms.gle/EpWTSWdfnVy6ToY98
```
Comment (cansavvy) [inline _data/navigation.yml:13]: This form probably needs more fleshing out before we make it live.

Comment (vortexing) [reply _data/navigation.yml:13]: Don't delete News - it's the blog. Jeff should weigh in on the avenue to get feedback from folks too.

**Review by vortexing — COMMENTED** [review]
Can you leave the training page called "training" and put that back into the header?  It's linked to in the Wiki and so renaming will break that.

File: _data/navigation.yml (hunk 1 above)
Comment (cansavvy) [reply _data/navigation.yml:13]: Yes, this is what I'm going to get Sean's review on.

Comment (cansavvy) [reply _data/navigation.yml:13]: News isn't the headline we called it in the Slack channel. I'm just trying to rectify that conversation with this.

Comment (vortexing) [reply _data/navigation.yml:13]: OH.  Well the website wires up a blog that gets rendered by news.md at /news/.  I'll go check Slack

**Comment by cansavvy** [comment]
> Can you leave the training page called "training" and put that back into the header? It's linked to in the Wiki and so renaming will break that.

Per this todo based on the Slack, we’re just doing big links to everything which includes the sciwiki so we don’t have to do much maintenance on this site.

https://public.3.basecamp.com/p/VjSeUbYe1b8XBcCddTuVqz85

File: _data/navigation.yml (hunk 1 above)
Comment (cansavvy) [reply _data/navigation.yml:13]: https://public.3.basecamp.com/p/VjSeUbYe1b8XBcCddTuVqz85 Check this specifically. It’s me combining Sean and Jeff’s thoughts

**Comment by cansavvy** [comment]
Also sorry, meant to mark this as draft because it’s not really meant for full review yet. 

I can request your review when it’s ready but first I need to get some of Sean’s input but I haven’t typed out my requests for his review yet

**Comment by vortexing** [comment]
I meant I have a PR to the WIki repo that links out to THIS site, but assumed the training page was just called "training".  I put my specific suggestions in a branch off your branch so you can see them.  Forms for contact us is unwise initially at the Hutch, so Jeff said in the immediate term, we should instead focus on people emailing us.  Also, given the clinical component, we definitely want whatever people might write to go into Hutch email.

**Comment by cansavvy** [comment]
I wouldn't bother doing things with this pull request yet. It's not ready.

**Comment by cansavvy** [comment]
But email is an unmanageable system.

**Comment by cansavvy** [comment]
> I wouldn't bother doing things with this pull request yet. It's not ready. 

That's what I meant before when I said I meant to have this as a draft PR.

File: _data/navigation.yml (hunk 1 above)
Comment (cansavvy) [reply _data/navigation.yml:13]: I think it makes sense to leave off a link to the blog until we actually have a first blog post.

Comment (cansavvy) [reply _data/navigation.yml:13]: What I'm doing is leaving the news.md file but I'm not going to link it in the navbar until it has something.

**Comment by cansavvy** [comment]
Retiring this PR in favor of #21

**Pull request closed without merging**